src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -110,7 +113,10 @@ protected override void BeginProcessing()
                 e is SecurityException
             )
             {
-                Exception exception = new Exception(TestJsonCmdletStrings.JsonSchemaFileOpenFailure, e);
+                Exception exception = new Exception(string.Format(
+                    CultureInfo.CurrentUICulture,
+                    TestJsonCmdletStrings.JsonSchemaFileOpenFailure,
+                    resolvedpath), e);
                 ThrowTerminatingError(new ErrorRecord(exception, "JsonSchemaFileOpenFailure", ErrorCategory.OpenError, null));
```

Comment:
You can use follow pattern:
```c#
                    var errorRecord = new ErrorRecord(
                        ex,
                        "FileNotFound",
                        ErrorCategory.ObjectNotFound,
                        path);
```

Reply:
So you suggest that I pass `resolvedpath` as a `targetObject` to the `ErrorRecord` constructor, so we additionaly get file path in the `$Error` ?

```csharp
ThrowTerminatingError(new ErrorRecord(exception, "JsonSchemaFileOpenFailure", ErrorCategory.OpenError, resolvedpath));
...
ThrowTerminatingError(new ErrorRecord(exception, "InvalidJsonSchema", ErrorCategory.InvalidData, resolvedpath));
```

```none
Exception    : System.Exception: Can not open JSON schema file: x:\y\invalid_schema_reference.json
...
TargetObject : x:\y\invalid_schema_reference.json <-- HERE ?
```
Makes sense. I'll do it for both `JsonSchemaFileOpenFailure` and `InvalidJsonSchema` `ErrorRecord`'s then?

Reply:
It is common pattern. You can look other cmdlet code.

Reply:
Thanks, done.